### DISCUSSION TITLE
Show folder location of file in results

### DISCUSSION THREAD

**Discussion by mouseman85** [microsoft-search/pnp-modern-search#2549] [General]
Is there a way to show just the parent directory name of where a file is stored and not the entire path? We have a library of help guides and i'd like to show where the file is located in the document library folders but not show the entire path so they know what category the file comes under without having to apply metadata tags.

**Comment by wobba** [comment]
Look at the ParentLink managed property, split on '/' and take the last part. That's the best you can do afaik.

**Reply by mouseman85** [reply]
Thanks Wobba, how do you go about splitting to get this to show up in a column or a card field? Would this be done outside PnP as a column or is it something that can be achieved with PnP?

**Reply by wobba** [reply]
Explore template helper options :) https://github.com/helpers/handlebars-helpers#helpers
